tests/test_pages.py

```diff
@@ -430,3 +436,151 @@ def nextPage(self):
         assert request.metadata is not None
         assert request.metadata.probability == 1.0
     assert type(page.nextPage) == Request
+
+
+def test_price_selector():
+    class CustomProductPage(ProductPage):
+        @field
+        def price(self):
+            return self.css("div::text")
+
+    html = b"""
+    <!DOCTYPE html>
+    <html>
+        <body>
+            <div>$13.2</div>
+        </body>
+    </html>
+    """
+    url = RequestUrl("https://example.com")
```

Comment:
I think you can pass a string directly (though it won't probably allow to save lines of code here)